mods/cnc/rules/world.yaml

```diff
@@ -6,6 +6,7 @@
 	MusicPlaylist:
 		VictoryMusic: win1
 		DefeatMusic: nod_map1
+	DebugVisualizations:
```

Comment:
It would probably make sense to add this via an upgrade rule. What to you think?